docs/cmdline_migration.md

```diff
@@ -0,0 +1,73 @@
+| [`-Xbootclasspath`](xbootclasspath.md)                | Specifies the search path for bootstrap classes and resources.                             |
+| [`-Xcheck:jni`](xcheck.md)                            | Runs additional checks for JNI functions during VM startup.                                |
+| [`-Xfuture`](xfuture.md)                              | Turns on strict class-file format checks.                                                  |
+| [`-Xint`](xint.md)                                    | Runs an application in interpreted-only mode.                                              |
+| [`-Xmn`](xmn.md)                                      | Sets the initial and maximum size of the new area when using -Xgcpolicy:gencon.            |
+| [`-Xms`](xms.md)                                      | Sets the initial size of the heap.                                                         |
+| [`-Xmx`](xmn.md)                                      | Specifies the maximum size of the memory allocation pool. (Equivalent to -XX:MaxHeapSize.) |
+| [`-Xnoclassgc`](xclassgc.md)                          | Disables class garbage collection (GC).                                                    |
+| [`-Xrs`](xrs.md)                                      | Prevents the OpenJ9 run time environment from handling signals.                            |
+| [`-Xss`](xss.md)                                      | Sets the thread stack size. (Equivalent to -XX:ThreadStackSize.)                           |
+| [`-Xverify:mode`](xverify.md)                         | Enables or disables the verifier.                                                          |
+| [`-XX:+DisableExplicitGC`](xxdisableexplicitgc)       | Disables System.gc() calls                                                                 |
+| [`-XX:-DisableExplicitGC`](xxdisableexplicitgc)       | Reenables System.gc() calls                                                                |
+| [`-XX:MaxDirectMemorySize`](xxmaxdirectmemorysize.md) | Sets a limit on the amount of memory that can be reserved for all direct byte buffers.     |
+| [`-XX:-UseCompressedOops`](xxusecompressedoops.md)    | Disables compressed references in 64-bit JVMs. (See also -Xcompressedrefs)                 |
+
+- These Hotspot command-line options have equivalents in OpenJ9 that are not specified in the same way, but perform the same function:
+
+| HotSpot Option          | OpenJ9 Option                                  | Usage                                                      |
+|-------------------------|------------------------------------------------|------------------------------------------------------------|                                                                        
+| `-Xcomp`                | [`-Xjit:count=0`](xjit.md)**<sup>1</sup>**     | `-Xcomp` disables interpreted method invocations.          |
```

Comment:
OpenJ9 understands -Xcomp and maps it to -Xjit:count=0. Did somebody request this option to be deprecated?